src/php/lib/Grpc/AbstractCall.php

```diff
@@ -68,25 +77,27 @@ public function __construct(Channel $channel,
         $this->deserialize = $deserialize;
         $this->metadata = null;
         $this->trailing_metadata = null;
-        if (isset($options['call_credentials_callback']) &&
+        if (array_key_exists('call_credentials_callback', $options) &&
```

Comment:
@thinkerou we need to check index in array, not a null value. I think it's more readable and clearly. Isn't it?